release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -2736,3 +2736,45 @@ link:https://access.redhat.com/solutions/5909931[{product-title} 4.6.23 containe
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.6 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI] for instructions.
+
+[id="ocp-4-6-24"]
+=== RHBA-2021:1011 - {product-title} 4.6.24 bug fix and security update
+
+Issued: 2021-04-06
+
+{product-title} release 4.6.24, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2021:1011[RHBA-2021:1011] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2021:1012[RHBA-2021:1012] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/5920851[{product-title} 4.6.24 container image list]
+
+[id="ocp-4-6-24-features"]
+==== Features
+
+[id="ocp-4-6-24-installing-cluster-on-vmc-on-aws"]
+===== Installing a cluster on VMC on AWS
+You can now install an {product-title} cluster on VMware vSphere infrastructure by deploying it to VMware Cloud (VMC) on AWS. See the documentation for xref:../installing/installing_vmc/installing-vmc.adoc#installing-vmc[deploying a cluster to VMC] for more information.
+
+[id="ocp-4-6-24-gathering-sap-backport"]
+===== Insights Operator enhancement for unhealthy SAP pods
+The Insights Operator can now gather data for unhealthy SAP pods. When the SDI installation fails, it is possible to detect the problem by looking at which of the initialization pods have failed. The Insights Operator now gathers  information about failed pods in the SAP/SDI namespaces.
+
+[id="ocp-4-6-24-bug-fixes"]
+==== Bug fixes
+
+* Previously, the file `/etc/pki/ca-trust/extracted` sometimes became unwritable, preventing the Image Registry Operator from adding CA certificates to the pod's trust store. With this release, an emptyDir volume is mounted into `/etc/pki/ca-trust/extracted` and the volume is now always writable by the pod. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1936984[*BZ#1936984*])
+
+* Previously, the automatically generated docker config secret did not include credentials for integrated internal registry routes.Because no credentials for accessing the registry through any of its routes were present, pods that attempted to reach the registry failed due to lack of authentication. With this release, all configured registry routes to the default docker credential secret are included, and pods can reach the integrated registry by any of its routes. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1931857[*BZ#1931857*])
```

Comment:
s/automatically generated/automatically-generated ??
Should _docker_ be capitalized? 
Space between _routes._ and _Before_